src/main/resources/static_config/static_roles.yml

```diff
@@ -85,6 +85,8 @@ kibana_server:
   - "cluster:admin/xpack/monitoring*"
   - "indices:admin/template*"
   - "indices:data/read/scroll*"
+  - 'cluster:admin/opendistro/reports/*'
```

Comment:
Why kibana_server role needs this permission?

Reply:
Kibana reporting plugin runs a background thread to poll job from ES reporting plugin. The Kibana executes the job in background and sends notification. It also updates status of the reports.

Reply:
And I believe we are planning to use kibanaserver user (which is mapped to kibana_server role) ? 

What exactly these two permissions are ?
'cluster:admin/opendistro/reports/*'
'cluster:admin/opendistro/notifications/*'

is there any other option to avoid using kibanaserver user?

Reply:
Also please keep formatting same and use double quote instead single.

Reply:
> And I believe we are planning to use kibanaserver user (which is mapped to kibana_server role) for running those background job ?
> 
> What exactly these two permissions are ? Can you describe more in details?
> 'cluster:admin/opendistro/reports/_' 'cluster:admin/opendistro/notifications/_'
> 
> is there any other option to avoid using kibanaserver user?

Would it be better to add a new user and grant the permission to it?

Reply:
Yes. kibanaserver is used for running background job.
cluster:admin/opendistro/notifications/* --> Send mail operation using notification plugin(currently there is only 1 operation).
cluster:admin/opendistro/reports/ --> all operations of reports plugin (currently we are using get_all, get, update, poll operations)

We are open to suggestion on using any system user for background operation. We cannot use the user role as until we pick the job we don't know which user is that job for.

Will update quite ' --> " and move before indices:admin

Reply:
is it preferred to explicitly add all used operations or use "*"?

Reply:
I would really prefer to use different user for background jobs instead piggybacking on kibanaserver user. 
You can have that user in reporting/notification configuration and if needed we can also add it into kibana config  

kibanaserver is just for kibana to run healthcheck and other calls for kibana UI. And hence kept it with very minimal set of permissions. 
Additionally, due to security threat associated with kibanaserver user, we want to keep permission very minimal to avoid any kibana user to gain benefit of kibanaserver user permission.  

Are reporting and notification plugins read 'kibanaserver' user and credential from kibana.yml and use it for background job? 
if yes, then can each plugin have their own user with its own default role with limited set of permission?  And can this user be defined in plugin configuration file ?  (I would prefer it into plugin yml file instead kibana.yml if possible)

Reply:
Are reporting and notification plugins read 'kibanaserver' user and credential from kibana.yml and use it for background job? --> NO. I believe kibana is already running with this user and no need to store/manage credential. Please let me know if my understanding is wrong.